GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -117,6 +117,7 @@ protected override void Execute(GVFSEnlistment enlistment)
 
                 if (!GVFSPlatform.Instance.KernelDriver.IsReady(tracer, enlistment.EnlistmentRoot, out errorMessage))
                 {
+                    tracer.RelatedInfo("ProjFs failed to start due to " + errorMessage);
                     tracer.RelatedInfo($"{nameof(MountVerb)}.{nameof(this.Execute)}: Enabling and attaching ProjFS through service");

```

Comment:
You could do a single `tracer` call here like this:

```
tracer.RelatedEvent(
    EventLevel.Informational, 
    $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService", 
    new EventMetadata
    {
        { "KernelDriver.IsReady_Error", errorMessage }
    });

Reply:
And if you want to include a message you can just add it to the EventMetadata:

`{ TracingConstants.MessageKey.InfoMessage, "Your message here" }`

Reply:
I added a message for clarity, so the line reads: 

[2018-10-25 11:06:47 -04:00] MountVerb_Execute_EnablingKernelDriverViaService {"KernelDriver.IsReady_Error":"Attaching the filter driver resulted in: 2147942405","Message":"Service will retry"}